src/pint/models/timing_model.py

```diff
@@ -1744,6 +1756,8 @@ def designmatrix(self, toas, acc_delay=None, incfrozen=False, incoffset=True):
             Whether to include frozen parameters in the design matrix
         incoffset : bool
             Whether to include the constant offset in the design matrix
+        incnoise : bool
```

Comment:
Will it ever be a good idea to include noise parameters in the design matrix? Is there a reason to create this non-functional parameter?

Reply:
I did it this way to ensure that the default behavior of PINT remains unchanged so that nothing else breaks.

Reply:
I think this parameter should be removed, and noise parameters not included. Any code that expects them to be included is currently broken, since none of them have correct derivatives.

Reply:
I'll remove the incnoise parameter.